cupidcoder/questioner
# #163145487 Set up database migrations and integrate database with Travis

#### What does this PR do?
- Configure database migration CLI executable and setting up Travis database integration

#### Description of Task to be completed?
- Wrote SQL queries for creating and dropping relevant tables at db/queries.js
- Added pg, dotenv and make-runnable npm packages
- Wrote npm scripts to allow for database migration automation
- Created migration and seeder files

#### How should this be manually tested?
- clone project to your local computer using the command
> git clone https://github.com/cupidcoder/questioner.git
- cd into the project local-test-migratedirectory and switch to branch using
> git checkout ch-setup-relevant-databases-163145487
- Run `npm install` to install relevant packages
- Create the database `questioner_test_db` in postgres
- Create your own .env file using the .env.example and add your own `TEST_DATABASE_URL` string
- Run the command `npm run local-test-migrate` and confirm from psql that the tables have been created

#### What are the relevant pivotal tracker stories?
- #163145487
